src/test/java/ru/training/at/hw1/CalculatorAddOperationUnitTest.java

```diff
@@ -0,0 +1,28 @@
+package ru.training.at.hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.*;
+
+import static org.testng.Assert.assertEquals;
+
+public class CalculatorAddOperationUnitTest {
+    private Calculator calculator = new Calculator();
+
+    @DataProvider(name = "data-provider")
+    public Object[][] createData() {
+        return new Object[][] {
+                { 2, 3, 5 },
+                { 0, 0, 0},
+                { -1, 1, 0}
+        };
+    }
+
+    @Test(groups = {"addSubtruct"}, dataProvider = "data-provider")
+    public void testAddLong(long v1, long v2, long result)  {
```

Comment:
Consider the use of more clear variables names (here and in other classes)
```suggestion
    public void testAddLong(long v1, long v2, long expected)  {
```

Reply:
fix it, and remove some unusefull code